<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fabuWidget</class>
 <widget class="QWidget" name="fabuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发布操作</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,6">
       <property name="leftMargin">
        <number>150</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择git</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>git分支</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gitTag</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="gitCombox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>150</number>
       </property>
       <property name="rightMargin">
        <number>150</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>git信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gitLine">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="cacelButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sureButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
